packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/representations/RepresentationLifecycleControllerIntegrationTests.java

```diff
@@ -149,7 +149,7 @@ public void givenRepresentationToRenameWhenMutationIsPerformedThenTheRepresentat
             assertThat(typename).isEqualTo(RenameRepresentationSuccessPayload.class.getSimpleName());
 
             assertThat(this.domainEventCollector.getDomainEvents()).hasSize(1);
-            assertThat(this.domainEventCollector.getDomainEvents()).anyMatch(RepresentationContentUpdatedEvent.class::isInstance);
+            assertThat(this.domainEventCollector.getDomainEvents()).anyMatch(RepresentationMetadataUpdateEvent.class::isInstance);
```

Comment:
We should be able to add back the check that I removed a looong time ago to ensure that it has been properly renamed